blackbird_python/blackbird/tests/test_load_dump.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the load(s) and dump(s) functions"""
+import sys
+from textwrap import dedent
+
+import pytest
+
+import numpy as np
+
+from blackbird import load, loads, dump, dumps
+
+
+test_script = """
+# a test blackbird script
+name test_name
+version 1.0
+target fock (num_subsystems=1, cutoff_dim=7, shots=10)

```

Comment:
Just curious, does blackbird require the space between the target and the brackets? What happens if it is not there?

Reply:
Nope, it doesn't require it, as I based the grammar off Python (where both `func(x)` and `func (x)` are identical grammar-wise).

The only reason I write it with a space is because it looks weird otherwise (`fock` isn't a function).